client/js/middleware/serializers/qbank/multiple_choice_serializer.js

```diff
@@ -0,0 +1,70 @@
+import _              from 'lodash';
```

Comment:
For whatever reason I got the serializer and deserializes mixed up. We probably need comments on both though, although then maintaining that becomes a royal pain. I'm thinking sorta similar to the canvas docs, just so you can be reasonably sure what the object is going to look like, without going and looking at the actual code that generates it.